templates/python39-mlops/requirements.txt

```diff
@@ -0,0 +1,6 @@
+algorithmia>=1.0.0,<2.0
+datarobot-drum==1.9.3
```

Comment:
james sorry if i missed anything, but do we need datarobot-drum?

Reply:
Good catch, removed unnecessary deps